CMakeLists.txt

```diff
@@ -296,6 +309,13 @@ add_executable(ufs_model NEMS/src/MAIN_NEMS.F90)
 add_dependencies(ufs_model ufs)
 target_link_libraries(ufs_model ufs esmf w3nco::w3nco_d)
 
+if(S2S OR CMEPS OR CDEPS)
+  if(PNETCDF)
+    target_link_libraries (ufs_model PRIVATE ${PNETCDF_LIBRARY_DIRS})
+    target_include_directories(ufs_model PRIVATE ${PNETCDF_INCLUDE_DIRS})
+  endif()
```

Comment:
I don't see anywhere `PNETCDF` is looked for in the `CMakeLists.txt`.
Also, does `PNETCDF` not provide imported targets?

Reply:
Yes, it seems strange. Maybe it is left over from one of my previous tests. I was testing PIO with PNETCDF support improving the performance of mediator history and restart as well as reading data through the CDEPS streams. Here is the more detail about that test (https://github.com/ESCOMP/CMEPS/pull/158, it is in the end of the thread). So, we might remove this part completely and test at least one coupled configuration.

Reply:
@aerorahul and @uturuncoglu, sure I will take out this code section accordingly and commit it back to the feature/hafs_sync_202107 branch. And will conduct regression tests after it being synced with ufs-weather-model develop again this afternoon.